javascript/javascript1/week3/smartApps.js

```diff
@@ -0,0 +1,85 @@
+const activities = [];
+let date = new Date();
+let day = date.getDate();
+let months = date.getMonth() + 1;
+let year = date.getFullYear();
+
+function getSum(arr) {
+  let counter = arr.reduce((a, b) => {
```

Comment:
When naming variables, try to choose names that explain what those variables are. Code that is easier to read and understand is easier to maintain.